Tests/SonarScanner.MSBuild.PreProcessor.Test/AnalysisConfigGeneratorTests.cs

```diff
@@ -384,6 +384,47 @@ public void GenerateFile_ExcludeCoverage_Exclusions_MultipleSpecified()
             .Which.Value.Should().Be("coverage1.xml,coverage2.xml,coverage3.xml,coverage4.xml");
     }
 
+    [DataTestMethod]
+    [DataRow("coverage.xml", "", "", "", "coverage.xml", "")]
+    [DataRow("coverage.xml", "", "local.cs,local.js", "", "local.cs,local.js,coverage.xml", "")]
+    [DataRow("coverage.xml", "", "", "server.cs,server.js", "", "server.cs,server.js,coverage.xml")]
```

Comment:
same as the other comment, I am not sure about emitting empty local exclusions ALWAYS.